Src/DependencyCollector/Shared.Tests/Implementation/ProfilerSqlProcessingTest.cs

```diff
@@ -361,6 +366,280 @@ public void OnEndLogsWarningWhenObjectPassedToEndWithoutCorrespondingBeginAsync(
             }
         }
 
+        [TestMethod]
+        public void FieldsForSqlConnectionAreCollectedCorrectly()
+        {
+            SqlConnection connection = GetSqlConnectionTest();
+            string actualResourceName = this.sqlConnectionProcessingProfiler.GetResourceName(connection);
+            Assert.AreEqual(ExpectedTarget, actualResourceName);
+
+            string actualTargetName = this.sqlConnectionProcessingProfiler.GetResourceTarget(connection);
+            Assert.AreEqual(ExpectedTarget, actualTargetName);
+
+            string actualCommandText = this.sqlConnectionProcessingProfiler.GetCommandName(connection);
+            Assert.AreEqual(ExpectedCommandTextForSqlConnection, actualCommandText);
+        }
+
+        [TestMethod]
+        public void OnEndAsyncSendsCorrectTelemetry_1ArgumentOverride()
+        {
+            var command = GetSqlCommandTestForStoredProc();
```

Comment:
I think the method name is great, no complaints about the method name.
I just don't know what "command" is, or what "context" is, etc.  There is no intellisense in GitHub.